package-lock.json

```diff
@@ -5412,6 +5317,88 @@
         }
       }
     },
+    "phantomjs-prebuilt": {
+      "version": "2.1.14",
```

Comment:
It appears that our `.travis.yaml` file specifies the OS to be `linux`, so I am not sure if we check different versions of OSX. Curious if we should update the script to include different OS versions?

It appears to be pretty straightforward:
https://docs.travis-ci.com/user/reference/osx/#OS-X-Version